### PR TITLE
examples: Update cluster example

### PR DESCRIPTION
**Pull request by nhlfr**
awstpr and clutertpr added more detailed fields for configuring Calico, Kubernetes and the Docker images used for deployment.

### CODE REVIEW

File: examples/cluster.yml [hunk 1]
```diff
@@ -10,10 +10,35 @@ spec:
...
+
+    calico:
+      subnet: "192.168.0.0"
+      cidr: "24"
+      mtu: "1500"
+
+    kubernetes:
+      api:
+        domain: "k8s.cluster.giantswarm.io"
+        insecurePort: "8080"
```
Comment (JosephSalisbury) [inline examples/cluster.yml]: I get json errors here, I think it needs to be just a plain int, as well as the line below

Comment (nhlfr) [inline examples/cluster.yml]: Good point, fixed that. I will also change MTU to int in `clustertpr` for better consistency.

File: examples/cluster.yml [hunk 2]
```diff
@@ -10,10 +10,35 @@ spec:
...
+      imageNamespace: "giantswarm"
+
+    etcd:
+      domain: "k8s.cluster.giantswarm.io"
+      prefix: "example-cluster"
+
+    calico:
+      subnet: "192.168.0.0"
+      cidr: "24"
+      mtu: "1500"
```
Comment (asymmetric) [inline examples/cluster.yml]: Should this be unquoted because of https://github.com/giantswarm/clustertpr/pull/8?

Comment (nhlfr) [inline examples/cluster.yml]: done

**Pull request merged**